test/math/FloatComplexTest.ooc

```diff
@@ -61,7 +61,6 @@ FloatComplexTest: class extends Fixture {
 			expect((complexNumber2 / complexNumber0) real, is equal to(2.6f) within(tolerance))
 			expect((complexNumber2 / complexNumber0) imaginary, is equal to(0.2f) within(tolerance))
 		})
-		// TODO: Add tests for compound assignment operators
```

Comment:
We don't test operators for most classes/covers, which is a problem, but it's not specific to `FloatComplex`. I will create a more general issue.